build.md

```diff
@@ -57,6 +57,19 @@ Install the build dependencies.
   * Build the binary.
   * Build the docker image with the binary.
 
+* To build multi-arch images, you need to have `buildx` installed. You can follow these [instructions](https://github.com/docker/buildx#installing).`
+  * NDM multi-arch images are built using QEMU mode. If you're on linux, run :
+    * docker run --rm --privileged linuxkit/binfmt:v0.8
+  * Set the value of envirionment variable of `IMAGE_ORG` based on your user name at Dockerhub:  
+    * export IMAGE_ORG=name
+  * Depending on what you're working on, run:
+    * For node-disk-manager:
+      * make docker.buildx.ndm
+    * For node-disk-operator:
+      * make docker.buildx.ndo
+    * For node-disk-exporter:
+      * make docker.buildx.exporter
+
```

Comment:
@harshthakur9030  Can you also provide a note that building multiarch image can take a lot of time since the build runs in a QEMU emulated environment

Reply:
Also add to the note, the list of currently supported architectures: (amd64, arm64, armv7, ppc64le)